reviewrot/pagurestack.py

```diff
@@ -123,7 +123,7 @@ def request_reviews(self, user_name, repo_name=None, state_=None,
                                url=url,
                                time=date,
                                comments=len(res['comments']),
-                               image=self._avatar(res['user']['name']),
+                               image=self._avatar(res['user']['name'], ssl_verify=ssl_verify),
```

Comment:
`PagureReview` seems to just inherit from `BaseReview` but I don't see `ssl_verify` as a keyword argument in the constructor.

Reply:
The parameter is going to the `_avatar` call, not `PagureReview.__init__`.

Reply:
Ah yeah, I misread that.